homeassistant/components/mcp_server/strings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "The Model Context Protocol (MCP) Server  integration exposes the Home Assistant Assist API to LLM clients on your network. MCP is an open protocol that standardizes how applications provide context to Large Language Models.\n\nThe Home Assistant MCP server requires authentication, such as a long lived access token. See the integration documentation for [more info]({more_info_url}).",
+        "data": {
+          "llm_hass_api": "[%key:common::config_flow::data::llm_hass_api%]"
+        },
+        "data_description": {
+          "llm_hass_api": "The method for controling Home Assistant to expose with the Model Context Protocol."
```

Comment:
Should we translate this as "Control", as we do for LLMs?

```suggestion
          "llm_hass_api": "Control"
```

(btw, I guess no control means really not much to be done)

Reply:
We're using `[%key:common::config_flow::data::llm_hass_api%]` above which is Control. I didn't see tha tothers have a `data_description` field. Just to make sure: Are you making this comment for the `data` field or `data_description` field?